src/componets/Calculator.js

```diff
@@ -1,22 +1,58 @@
 import React from 'react';
 import classes from './Home.module.css';
 import Button from './Button';
+import calculate from '../logic/calculate';
 
-export default function Calclator() {
-  const buttons = [
-    'AC', '+/-', '%', '+', '7', '8', '9', 'X', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=',
-  ];
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      obj: {},
+    };
+    this.btn = '';
+  }
 
-  return (
-    <div className={classes.home}>
-      <div className={classes.inner}>
-        <div className={classes.result}>
-          <div className={classes.resbox} />
-        </div>
-        <div className={classes.btns}>
-          {buttons.map((value) => <Button value={value} key={value} />)}
+  handleClick = (btn) => {
+    const { obj } = this.state;
+    this.setState({ obj: calculate(obj, btn) });
+    this.btn = btn;
+  };
+
+  showData = (obj) => {
+    if (obj.next === 'AC') {
+      return 0;
+    }
+
+    if (this.btn === '=') {
+      return obj.total;
+    }
+    return obj.next;
+  }
+
+  render() {
+    const buttons = [
+      'AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=',
+    ];
+    const { obj } = this.state;
```

Comment:
- [ ] Hey! 👋 Just wanted to share a suggestion with you. When someone taps the A/C button, it'd be awesome if all the input gets cleared. This way, your app's user experience could become even smoother. Keep up the great work! You're doing awesome! 💪😄